projects/1.20-fabric/assets/chatpatches/chatpatches/lang/zh_cn.json

```
@@ -0,0 +1,136 @@
+{
+	"modmenu.descriptionTranslation.chatpatches": "一个多功能的客户端聊天模组，让聊天更加定制化，查看配置和GitHub页面了解更多信息！\n我们的Discord是https://discord.gg/3mqbvneymz！",
+
+	"text.chatpatches.title": "Chat Patches 配置",
+
+	"text.chatpatches.help.dateFormat": "关于日期和时间格式的信息，点击这里！",
+	"text.chatpatches.help.formatCodes": "关于格式化代码字符以及各种功能的信息，点击这里！",
+	"text.chatpatches.help.faq": "常见问题解答 (FAQ)，点击这里！",
+	"text.chatpatches.help.regex": "关于正则表达式的信息，点击这里！",
+	"text.chatpatches.help.regexTester": "正则表达式测试工具，点击这里！",
+
+	"text.chatpatches.category.time": "时间戳",
+	"text.chatpatches.category.hover": "悬停信息",
+	"text.chatpatches.category.counter": "重复计数",
+	"text.chatpatches.category.counter.compact": "§9CompactChat§r 重复计数",
+	"text.chatpatches.category.boundary": "会话边界",
+	"text.chatpatches.category.chatlog": "聊天记录",
+	"text.chatpatches.category.chatlog.actions": "行动",
+	"text.chatpatches.category.chat": "聊天界面",
+	"text.chatpatches.category.chat.hud": "HUD（悬浮显示）",
+	"text.chatpatches.category.chat.screen": "屏幕",
+	"text.chatpatches.category.copy": "复制菜单",
+	"text.chatpatches.category.help": "帮助",
+	"text.chatpatches.category.debug": "调试",
+	"text.chatpatches.category.desc.counter.compact": "关于更具“操作性”的聊天重复方法的选项，基于 §9§nCompactChat§r 模组。",
+	"text.chatpatches.category.desc.chat": "与聊天 HUD、聊天界面或消息相关的所有内容。",
+	"text.chatpatches.category.desc.chat.hud": "关于即使聊天界面未打开时也会显示的小部件的选项。这也可以包括与整个聊天相关的设置。",
+	"text.chatpatches.category.desc.chat.screen": "特定于屏幕的选项。",
+	"text.chatpatches.category.desc.copy": "关于在聊天消息上右键单击时出现的选择框和复制菜单的选项。",
+	"text.chatpatches.category.desc.help": "解释如何使用某些模组方面的网站！",
+	"text.chatpatches.category.desc.debug": "用于开发的随机内容？",
+
+	"text.chatpatches.time": "时间戳开关",
+	"text.chatpatches.timeDate": "时间戳文本",
+	"text.chatpatches.timeFormat": "时间戳装饰文本",
+	"text.chatpatches.timeColor": "时间戳颜色",
+	"text.chatpatches.desc.time": "是否显示所有消息前面的时间戳？",
+	"text.chatpatches.desc.timeDate": "将格式化为时间戳的文本。",
+	"text.chatpatches.desc.timeFormat": "创建时间戳的字符串，其中的‘&’是格式代码修饰符和装饰文本。‘$’是前面此选项的格式化时间戳的占位符。",
+	"text.chatpatches.desc.timeColor": "填充颜色以用于结果中可能为空白的时间戳部分。要与其他格式修饰符一起使用此选项，请在装饰文本选项中使用‘&r’。",
+
+	"text.chatpatches.hover": "悬停开关",
+	"text.chatpatches.hoverDate": "悬停文本",
+	"text.chatpatches.hoverFormat": "悬停装饰文本",
+	"text.chatpatches.hoverColor": "悬停文本颜色",
+	"text.chatpatches.desc.hover": "当你悬停在聊天中的时间戳文本上时，是否显示一个更详细的时间描述？",
+	"text.chatpatches.desc.hoverDate": "将格式化为较长日期/时间字符串的文本。",
+	"text.chatpatches.desc.hoverFormat": "创建详细悬停描述的字符串，其中的‘&’是格式代码修饰符和装饰文本。需要一个‘$’作为前面此选项的格式化时间戳的占位符。",
+	"text.chatpatches.desc.hoverColor": "填充颜色以用于结果中可能为空白白色的悬停文本。要与其他格式修饰符一起使用此选项，请在装饰文本选项中使用‘&r’。",
+
+	"text.chatpatches.counter": "消息计数器开关",
+	"text.chatpatches.counterFormat": "消息计数器文本",
+	"text.chatpatches.counterColor": "消息计数器颜色",
+	"text.chatpatches.counterCompact": "CompactChat 方法开关",
+	"text.chatpatches.counterCompactDistance": "消息检查距离",
+	"text.chatpatches.desc.counter": "是否在消息后显示消息计数器以指示已发送多个重复消息？注意：要使 CompactChat 选项起作用，必须启用此选项。",
```

```suggestion
	"text.chatpatches.desc.counter": "是否在消息后显示消息计数器以指示已发送多个重复消息？注意：必须启用此选项以使CompactChat选项起作用。",
```
可不改，原文可能会有歧义，读成两个不相关的单句
要改的话后续的格式建议也一并改了